.circleci/config.yml

```diff
@@ -70,10 +70,21 @@ jobs:
           command: |
               if [ $CIRCLE_BRANCH = 'master' ]; then
                   ./gradlew :app:uploadDeployGateRelease
+              elif [[ "${CIRCLE_TAG}" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+                  echo "Deploy to Google Play"
+                  ./gradlew clean assembleRelease publishApkRelease
               fi
       - store_artifacts:
           path: app/build/reports
           destination: reports
       - store_artifacts:
           path: app/build/outputs/apk
           destination: apk
+
+# workaround for tag trigger
+# because workflow git tag trigger does not work well
+deployment:
+  fake_for_deploy:
+    tag: /.*/
```

Comment:
This is fake trigger for workaround.
Honestly I wanted to use CircleCI2 workflow git tag trigger feature, but it doesn't work well 😢